### ISSUE TITLE
Specify default math class in documentation

### ISSUE THREAD

**Issue by MDLC01** [typst/webapp-issues#175]
### Description

On the [General Symbols page](https://typst.app/docs/reference/symbols/sym/), I would like the default math class (which can be overridden with [`math.class`](https://typst.app/docs/reference/math/class/)) to be indicated when clicking on a character.

### Use Case

Knowing what the default math class of a character is easily.

**Comment by MDLC01** [comment]
Good news: this feature seems to have been implemented at https://staging.typst.app/docs/reference/symbols/sym/!

**Comment by laurmaedje** [comment]
Yeah, I think we can just close this. The docs changes will definitely go into the main version, so it's kind of like closing a bug in the dev compiler even though it's not released yet.
